presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -393,12 +393,18 @@ public void testColumnComment()
     {
         try (AutoCloseable ignore = withTable("tpch.test_column_comment",
                 "(col1 bigint, col2 bigint, col3 bigint)")) {
-            execute("COMMENT ON COLUMN tpch.test_column_comment.col1 IS 'test comment'");
-            execute("COMMENT ON COLUMN tpch.test_column_comment.col2 IS ''"); // it will be NULL, PostgreSQL doesn't store empty comment
+            assertUpdate("COMMENT ON COLUMN tpch.test_column_comment.col1 IS 'test comment'");
+            assertUpdate("COMMENT ON COLUMN tpch.test_column_comment.col2 IS ''"); // it will be NULL, PostgreSQL doesn't store empty comment
```

Comment:
> PostgreSQL doesn't store empty comment

Then maybe we should not allow users to set empty comments. Displaying nulls when empty string was set could be misleading.

Reply:
> > PostgreSQL doesn't store empty comment
> 
> Then maybe we should not allow users to set empty comments. Displaying nulls when empty string was set could be misleading.

arguably this ship has sailed -- we've deliberately followed this path in case of other connectors before.

Reply:
> arguably this ship has sailed -- we've deliberately followed this path in case of other connectors before.

What postgres do when user do `COMMENT ON COLUMN tpch.test_column_comment.col2 IS null`?
If it fails we should fail as well. I don't want to comment other connector behaviour.

Reply:
PostgreSQL removes a comment in case of `IS NULL`.
```bash
tpch=# \d+ tpch.test;
                          Table "tpch.test"
 Column |  Type   | Modifiers | Storage | Stats target | Description
--------+---------+-----------+---------+--------------+-------------
 c1     | integer |           | plain   |              | foo

tpch=#
tpch=# comment on column tpch.test.c1 is null;
COMMENT
tpch=#
tpch=# \d+ tpch.test;
                          Table "tpch.test"
 Column |  Type   | Modifiers | Storage | Stats target | Description
--------+---------+-----------+---------+--------------+-------------
 c1     | integer |           | plain   |              |
```

https://www.postgresql.org/docs/current/sql-comment.html
> To remove a comment, write NULL in place of the text string

Reply:
What does it mean `remove a comment`? Insert `null`?

What about
```
SELECT d.description IS NULL FROM pg_description d, pg_class c WHERE  d.oid = c.oid AND c.relname = 'test'
```
?

Reply:
Yes, it inserts `null`. 
```bash
tpch=# \pset null (null)
Null display is "(null)".
tpch=#
tpch=# comment on column tpch.test.c1 is 'foo';
COMMENT
tpch=#
tpch=# select c.column_name, c.data_type, pgd.description, pgd.description is null
tpch-# from pg_catalog.pg_statio_all_tables as st
tpch-# inner join pg_catalog.pg_description pgd on (pgd.objoid=st.relid)
tpch-# right outer join information_schema.columns c on (pgd.objsubid=c.ordinal_position and  c.table_schema=st.schemaname and c.table_name=st.relname)
tpch-# where table_schema = 'tpch' and table_name = 'test';
 column_name | data_type | description | ?column?
-------------+-----------+-------------+----------
 c1          | integer   | foo         | f
(1 row)

tpch=#
tpch=# comment on column tpch.test.c1 is null;
COMMENT
tpch=#
tpch=# select c.column_name, c.data_type, pgd.description, pgd.description is null
tpch-# from pg_catalog.pg_statio_all_tables as st
tpch-# inner join pg_catalog.pg_description pgd on (pgd.objoid=st.relid)
tpch-# right outer join information_schema.columns c on (pgd.objsubid=c.ordinal_position and  c.table_schema=st.schemaname and c.table_name=st.relname)
tpch-# where table_schema = 'tpch' and table_name = 'test';
 column_name | data_type | description | ?column?
-------------+-----------+-------------+----------
 c1          | integer   | (null)      | t
```
(Changed query because it didn't return anything)